testgrid/specs/full.yaml

```diff
@@ -1679,6 +1679,62 @@
 
     k8sVersion=$(kubectl get nodes -o jsonpath='{.items[0].status.nodeInfo.kubeletVersion}')
     echo $k8sVersion | grep 1.20
+- name: rook-upgrade-to-1.5-airgap
+  flags: "yes"
+  airgap: true
+  installerSpec:
+    kubernetes:
+      version: 1.19.x
+    containerd:
+      version: "latest"
+    weave:
+      version: "latest"
+    rook:
+      version: "1.0.4"
+    prometheus:
+      version: "latest"
+    minio:
+      version: "latest"
+  upgradeSpec:
+    kubernetes:
+      version: 1.19.x
+    containerd:
+      version: "latest"
+    weave:
+      version: "latest"
+    rook:
+      version: "1.5.12"
+    prometheus:
+      version: "latest"
+    minio:
+      version: "latest"
+  preInstallScript: |
+    # download the file that the airgap upgrade requires, and place it in the correct location
+    curl -LO https://kurl-sh.s3.amazonaws.com/staging/rookupgrade-10to14.tar.gz
+    mkdir -p /var/lib/kurl/assets
+    mv rookupgrade-10to14.tar.gz /var/lib/kurl/assets
+  postInstallScript: |
+    source /opt/kurl-testgrid/testhelpers.sh
+    rook_ecph_object_store_info
+    validate_read_write_object_store rwtest testfile.txt
+
+    minio_object_store_info
+    validate_read_write_object_store rwtest minio.txt
+  postUpgradeScript: |
+    source /opt/kurl-testgrid/testhelpers.sh
+    rook_ecph_object_store_info
```

Comment:
Nit: Looks like there's a typo in the script name: `rook_ecph_...` instead of `rook_ceph_...`

Reply:
1. yes, there is a typo in the name
2. unfortunately, I didn't catch that [before committing the script](https://github.com/replicatedhq/kURL-testgrid/blob/e34f686743c37f8a7fc6d5d6cc25a5af0298d733/tgrun/pkg/runner/vmi/embed/testhelpers.sh#L85) a few months ago and haven't found it worth it to fix since then